src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updateSubmission/UpdateSubmission.java

```diff
@@ -0,0 +1,130 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updateSubmission;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntitySubmissionDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.updateSubmission.changes.Change;
+
+import java.util.List;
+
+public class UpdateSubmission {
+    @JsonProperty("type")
+    private String type = "OE02";
+
+    @JsonProperty("userSubmission")
+    private OverseasEntitySubmissionDto userSubmission;
+
+    @JsonProperty("dueDiligence")
+    private DueDiligence dueDiligence;
+
+    @JsonProperty("presenter")
+    private Presenter presenter;
+
+    @JsonProperty("filingForDate")
+    private FilingForDate filingForDate;
+
+    @JsonProperty("noChangesInFilingPeriodStatement")
+    private String noChangesInFilingPeriodStatement;
+
+    @JsonProperty("anyBOsOrMOsAddedOrCeased")
+    private String anyBOsOrMOsAddedOrCeased;
+
+    @JsonProperty("beneficialOwnerStatement")
+    private String beneficialOwnerStatement;
```

Comment:
Might need to incorporate 2 statements -- please confirm this

Reply:
`<pscStatement>
        <pscStatementType/>
    </pscStatement>`

Reply:
I see two here. But they should be booleans.